atomic_reactor/util.py

```diff
@@ -912,6 +914,26 @@ def _config_by_digest(self, image: ImageName, config_digest: str) -> dict:
         blob_config = config_response.json()
         return blob_config
 
+    def _config_and_id_from_manifest_list(
+        self, image: ImageName, manifest_list: dict, arch: Optional[str]
+    ) -> Tuple[dict, str]:
```

Comment:
How about just pass `manifest_list["manifests"]` directly?

Reply:
I would rather keep it as it is. `manifest_list["manifests"]` is a detail of the [manifest list](https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list) format, currently only the `_config_and_id_from_manifest_list` method has to understand those details